.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           working_directory: tmconfig/
           command: |
             IMAGE_REPO=$(sed -n -e 's/^IMAGE_REPO[[:space:]].*=[[:space:]]*\(.*\)$/\1/p' ~/project/Makefile)
-            for cmd in $(sed -n -e 's/^COMMANDS[[:space:]]*=[[:space:]]*\(.*\)$/\1/p' ~/project/Makefile); do
+            for cmd in $(ls ~/project/cmd/); do
```

Comment:
That pattern is usually considered [fragile](https://github.com/koalaman/shellcheck/wiki/SC2045).

Here is a suggestion:

```sh
for cmd in cmd/*; do
    echo ${cmd##*/}       # option 1
    echo $(basename $cmd) # option 2
done
```

Reply:
updated! and applied the same to hack/release-notes.sh